spring-xd-dirt/src/main/java/org/springframework/xd/dirt/modules/metadata/FileAsRefMixin.java

```diff
@@ -44,6 +46,15 @@ public void setMode(FileReadingMode mode) {
 		this.fileReadingmode = mode;
 	}
 
+	public boolean getSplitWithMarkers() {
+		return splitWithMarkers;
+	}
+
+	@ModuleOption("if true emits start of file/end of file marker messages before/after the data. Defaults to false. Only valid with FileReadingMode 'lines'")
```

Comment:
No need to specify the default; it's obtained from the field value (see the doc below).

Also, we can add an `@AssertTrue` method with text to ensure the property is only set with `lines` mode.

See `RedisSinkOptionsMetadata` for an example.